core/trino-main/src/main/java/io/trino/operator/scalar/StringFunctions.java

```diff
@@ -968,4 +969,67 @@ public static Slice soundex(@SqlType(StandardTypes.VARCHAR) Slice slice)
             throw new TrinoException(INVALID_FUNCTION_ARGUMENT, e);
         }
     }
+
+    @Description("Returns Fuzzy Match Simple Ratio")
```

Comment:
Please consider adding corresponding documentation for the syntax of the newly added string functions within `string.rst` file (so that the functions get eventually listed here: https://trino.io/docs/current/functions/string.html)